apps/gp2-frontend/src/onboarding/Routes.tsx

```diff
@@ -1,17 +1,24 @@
-import { BasicLayout, OnboardWelcome } from '@asap-hub/gp2-components';
+import { NotFoundPage } from '@asap-hub/react-components';
+import { gp2 } from '@asap-hub/routing';
 import { Route, Switch, useRouteMatch } from 'react-router-dom';
+import CoreDetails from './CoreDetails';
+import Welcome from './Welcome';
+
+const { onboarding } = gp2;
 
 const Routes: React.FC<Record<string, never>> = () => {
   const { path } = useRouteMatch();
 
   return (
-    <BasicLayout>
-      <Switch>
-        <Route exact path={path}>
-          <OnboardWelcome />
-        </Route>
-      </Switch>
-    </BasicLayout>
+    <Switch>
+      <Route exact path={path}>
+        <Welcome />
+      </Route>
+      <Route exact path={onboarding({}).coreDetails.template}>
```

Comment:
in this case it's gonna be the same but I like using the `onboarding({}).coreDetails({}).$` better

Reply:
Done